src/UserChipMenu/UserChipMenu.tsx

```diff
@@ -45,13 +45,7 @@ export const UserChipMenu: React.FC<UserChipProps> = ({
         state={isMenuOpened ? "active" : "default"}
         {...props}
       >
-        {avatar ? (
```

Comment:
don't we want to deprecate this? seems like a breaking change.

Reply:
The `avatar` prop is the URL to the users avatar image, here's an example from dashboard:

```tsx
    <UserChipMenu
      initials={getUserInitials(user)}
      name={getUserName(user, true)}
      avatar={user?.avatar?.url}
    >
```

in my opinion this is not breaking change, I've just refactored that part into a separate component so that we can use it in other places ;)

Reply:
ahh, you are right - you still render avatar but as a component, sorry didn't noticed 👍